server/src/main/java/org/elasticsearch/index/mapper/TimestampFieldMapper.java

```diff
@@ -196,6 +196,10 @@ public void validate(DocumentFieldMappers lookup) {
         }
     }
 
+    public String getPath() {
```

Comment:
Does `TimestampFieldMapper` need a `path` parameter now? We're not sure how we will eventually implement custom timestamp names, I can see an implementation where we don't use `path` here at all. So maybe we could simplify and just have an `enabled` boolean flag instead.

Depending on the implementation, we can always add `path` later and just default to `@timestamp` if it's not specified.